### ISSUE TITLE
Keyboard support?

### ISSUE THREAD

**Issue by codepo8** [arnaudleyder/magneticmediajs#2]
One of the main things missing is keyboard controls. ESC to close the gallery, arrows to navigate...

**Comment by arnaudleyder** [comment]
Hi Christian,

Thanks for your input. I have started to implement this feature (mainly for accessibility purposes). I can confirm it does not work in 1.3.2. Hopefully I will fix it for 1.3.3 which I hope will be soon.

Arnaud

**Comment by arnaudleyder** [comment]
Fix in commit fbac5e4d1c. Will push to https://www.magneticmediajs.com demo site when 1.3.3 has deployed on jsDelivr.
